proxy/http/HttpTransact.cc

```diff
@@ -567,6 +567,16 @@ HttpTransact::BadRequest(State *s)
 }
 
 void
+HttpTransact::Forbidden(State *s)
+{
+  DebugTxn("http_trans", "[Forbidden]"
+                         "parser marked request forbidden");
```

Comment:
I added this Forbidden state and copied the debug message from the BadRequest state